lib/graphql/account_settings/cloud_manager.dart

```diff
@@ -118,17 +146,22 @@ class CloudManager {
   }
 
   Future<void> downloadAll() async {
+    log.info('[CloudManager] downloadAll');
+    // lazy load for playlists
+    unawaited(injector<SettingsDataService>().restoreSettingsData());
```

Comment:
could you please the lazy load for playlist. We need playlist as soon as walletAddress.

Reply:
it is loaded in playlist service